frontend/src/lib/components/LemonActionableTooltip/LemonActionableTooltip.tsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react'
+import { Placement } from '@floating-ui/react-dom-interactions'
+import { Popup } from 'lib/components/Popup/Popup'
+import { IconOpenInNew } from 'lib/components/icons'
+import { CloseOutlined, LeftOutlined, RightOutlined } from '@ant-design/icons'
+import { LemonButton } from '@posthog/lemon-ui'
+import './LemonActionableTooltip.scss'
+
+export type LemonActionableTooltipProps = {
+    text: string
+    placement: Placement
+    step: number
+    maxSteps: number
+    visible: boolean
+    close: () => void
+    element?: HTMLElement
+    next?: () => void
+    previous?: () => void
+    buttons?: { label: string; url?: string; action?: () => void }[]
+    icon?: JSX.Element
+}
+
+export const LemonActionableTooltip = ({
```

Comment:
should we insist on storybook stories for all components?

**not blocking** since it could be added in a follow-up

Reply:
Good point, I will do the stories for these in a separate PR